libPhoneNumber/NBGeocoderMetadataHelper.h

```diff
@@ -0,0 +1,52 @@
+//
+//  NBGeocoderMetadataHelper.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <sqlite3.h>
+
+@class NBPhoneNumber;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NBGeocoderMetadataHelper : NSObject
+
+/**
+* Initializer method that creates a SQLite3 connection to the correct database (based on
```

Comment:
The * should be indented +1 to match the first * on the previous line. Applies everywhere

Reply:
fixed.